etc/cargo.config.prod

```diff
@@ -0,0 +1,14 @@
+# Configuration for release builds, that is bulids that are
+# _literally_ for packaging and releasing; and cases where we _must_
+# benchmark against a fully-optimized build.
+
+[profile.release]
+opt-level = 3
+debug = false
```

Comment:
TiKV may need debug info in order to display specific line when unwinding, or maybe can we generate debuginfo separate?

Reply:
Yeah, this is something I need to look at closer, though I think we are going to have to keep debug=true for release builds, but not for benchmarking builds. I've pushed a change to set debug=true.

We can't generate debuginfo separately.

Per https://github.com/tikv/tikv/issues/4202, where I expect to end up is distributing bins without debuginfo, but such that the debuginfo can be distributed on the side when needed. To do that though we have to have rustc generate the libraries with debuginfo, then strip the debuginfo.